test/UnitTests/MSTest.Core.Unit.Tests/Assertions/AssertTests.cs

```diff
@@ -245,5 +245,37 @@ public void ReplaceNullCharsShouldReplaceNullCharsInAString()
         }
 
         #endregion
+
+        #region InstanceOfType tests
+
+        [TestMethod]
+        public void InstanceOfTypeShouldFailWhenValueIsNull()
+        {
+            Action action = () => TestFrameworkV2.Assert.IsInstanceOfType(null, typeof(AssertTests));
+            TestFrameworkV2.Assert.ThrowsException<TestFrameworkV2.AssertFailedException>(action);
```

Comment:
Can we use [ActionUtility.ActionShouldThrowExceptionOfType](https://github.com/Microsoft/testfx/blob/5f8e5a421aa6a7f64f3aa458aebb418c2fd90ce6/test/UnitTests/MSTest.CoreAdapter.TestUtilities/ActionUtility.cs#L26) instead please. Any issue with ThrowsException would leave these tests in an indeterminate state too. Hence the ask.